eng/Packaging.props

```diff
@@ -23,12 +23,13 @@
     <SkipBaseLineCheck>true</SkipBaseLineCheck>
 
     <!-- by default all packages will use the same version which revs with respect to product version -->
-    <PackageVersion Condition="'$(PackageVersion)' == ''">4.6.0</PackageVersion>
+    <PackageVersion Condition="'$(PackageVersion)' == ''">4.6.1</PackageVersion>
     <!-- major.minor.release version of the platforms package we're currently building
          Pre-release will be appended during build -->
     <PlatformPackageVersion>$(ProductVersion)</PlatformPackageVersion>
     <SkipValidatePackageTargetFramework>true</SkipValidatePackageTargetFramework>
     <SkipGenerationCheck>true</SkipGenerationCheck>
+    <SkipIndexCheck>true</SkipIndexCheck>
```

Comment:
We should note that this is currently set to avoid errors that would result from having the 3.0 release packages not yet stable.  Once 3.0 ships we can add those packages to the index and re-enable this.